xml/src/test/resources/configs/resources-templates.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+  ~ Copyright Terracotta, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<ehcache:config
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:ehcache='http://www.ehcache.org/v3'
+    xsi:schemaLocation="http://www.ehcache.org/v3 ../../../main/resources/ehcache-core.xsd">
+
+  <ehcache:cache-template name="tieredResourceTemplate">
+    <ehcache:key-type>java.lang.String</ehcache:key-type>
+    <ehcache:value-type>java.lang.String</ehcache:value-type>
+    <ehcache:resources>
+      <ehcache:heap value="5"/>
+      <ehcache:disk value="50"/>
+    </ehcache:resources>
+  </ehcache:cache-template>
+
+  <ehcache:cache alias="templatedTieredResource" usesTemplate="tieredResourceTemplate">
+  </ehcache:cache>
+
+  <ehcache:cache-template name="explicitHeapResourceTemplate">
+    <ehcache:key-type>java.lang.String</ehcache:key-type>
+    <ehcache:value-type>java.lang.String</ehcache:value-type>
+    <ehcache:resources>
+      <ehcache:heap value="15"/>
+    </ehcache:resources>
+  </ehcache:cache-template>
+
+  <ehcache:cache alias="templatedExplicitHeapResource" usesTemplate="explicitHeapResourceTemplate">
+  </ehcache:cache>
+
+  <ehcache:cache-template name="implicitHeapResourceTemplate">
```

Comment:
Was it not commented yesterday that you would need at least the `<ehcache:heap value="5"/>` element if you do not have a `<ehcache:resources>` one?

As we said no resource configured means no resource available.